ccpp/CMakeLists.txt

```diff
@@ -96,10 +92,7 @@ endif()
 add_subdirectory(framework)
 
 add_subdirectory(physics)
-add_dependencies(ccppphys ccpp)
-target_link_libraries(ccppphys PUBLIC w3nco::w3nco_d NetCDF::NetCDF_Fortran)
-# This should not be necessary once framework and physics targets define BUILD_INTERFACE
-target_include_directories(ccppphys PRIVATE ${CMAKE_CURRENT_BINARY_DIR}/framework/src)
+#add_dependencies(ccpp_physics ccpp_framework) ! Not sure if this is needed at all
```

Comment:
If this line is not needed, then please remove it.

Reply:
done in a89df7b